msteams-platform/concepts/build-and-test/Shared-channels.md

```diff
@@ -0,0 +1,105 @@
+
+SupportedChannelTypes is an optional property that enables your app in non-standard channels. If your app supports the team scope and the property is defined, Teams will enable your app in each channel type accordingly. Private and shared channels are currently supported. For more information, see [supportedChannelTypes](../../resources/schema/manifest-schema.md#supportedchanneltypes)
+
+```JSON
+"supportedChannelTypes": {
+      "type": "array",
+      "description": "List of ‘non-standard’ channel types that the app supports. Note: Channels of standard type are supported by default if the app supports team scope. ",
+      "maxItems": 2,
+      "items": { 
+        "enum": [
+          "sharedChannels",
+          "privateChannels"
+        ]
+      }
+    }
+```
+
+> [!NOTE]
+>
+> * If your app supports the team scope, it will always function in standard channels, regardless of what values are defined in this property.
+> * Your app may need to account for the unique properties of each of these channel types in order to function properly.
+
+For more information on enabling your tab, see:
+
+* [Get context for your tab for private channels](../../tabs/how-to/access-teams-context.md#retrieve-context-in-private-channels)
+* [Get context for your tab for shared channels](../../tabs/how-to/access-teams-context.md#retrieve-context-in-microsoft-teams-connect-shared-channels)
+
+## Apps and permissions in shared channels
+
+You can collaborate with external members outside of your organization using shared channels. App permissions in shared channels follow the host team's app roster and host tenant's app policy.
+
+> [!NOTE]
+> The [activity feed notification API](/graph/teams-send-activityfeednotifications) doesn't support cross-tenant notifications for apps in a shared channel.
+
+## Get shared channel membership
+
+You can get direct shared channel membership by using the `hostTeamGroupID` from `getContext` and following these steps:
+
+1. Get direct members with [GET channel members API](/graph/api/channel-list-members?view=graph-rest-beta&tabs=http&preserve-view=true) API.
+
+    ```http
+    GET /teams/{host-team-group-id}/channels/{channel-id}/members
+    ```
+
+2. Get each shared team with GET `sharedWithTeams` API.
+
+    ```http
+    GET /teams/{host-team-group-id}/channels/{channel-id}/sharedWithTeams
+    ```
+
+3. Use GET members of each shared team (Team X) with GET `sharedWithTeams` API.
+
+    ```http
+    GET /teams/{host-team-group-id}/channels/{channel-id}/sharedWithTeams/{teamX}/members
+    ```
+
+## Classify members in the shared channel as in-tenant or out-tenant
+
+You can classify members as in-tenant or out-tenant by comparing `tenantID` of the member or team with `hostTeamTenantID` as follows:
```

Comment:
The use of `in-tenant` and `out-tenant` is almost like coining new terms here. While the former is used a few times on docs.microsoft but the latter is completely new. Why do we need a new term here? It seems we can just call those users with (or without) an account on a tenant.